docs/data/base/components/switch/UseSwitchesBasic.tsx

```diff
@@ -36,22 +36,33 @@ const blue = {
 };
 
 const grey = {
-  400: '#BFC7CF',
-  500: '#AAB4BE',
+  50: '#F3F6F9',
+  100: '#E7EBF0',
+  200: '#E0E3E7',
+  300: '#CDD2D7',
+  400: '#B2BAC2',
+  500: '#A0AAB4',
   600: '#6F7E8C',
+  700: '#3E5060',
+  800: '#2D3843',
+  900: '#1A2027',
 };
 
 const BasicSwitchRoot = styled('span')(
   ({ theme }) => `
   font-size: 0;
   position: relative;
   display: inline-block;
-  width: 40px;
+  width: 48px;
```

Comment:
```suggestion
  width: 38px;
```

I think we can make it shorter, actually!